packages/loader/driver-utils/src/messageRecognition.ts

```diff
@@ -12,18 +12,24 @@ import {
  * Tells if message was sent by container runtime
  * @privateRemarks ADO #1385: To be moved to container-definitions
  * @returns whether the message is a runtime message
+ * @public
  */
 export function isRuntimeMessage(message: { type: string }): boolean {
 	return message.type === MessageType.Operation;
 }
 
-// ADO #1385: staging code changes across layers.
-// Eventually to be replaced by MessageType.accept
+/** ADO #1385: staging code changes across layers.
```

Comment:
Nit: formatting feedback + recommendation since this is a comment for us, more than for consumers, I would recommend:
```suggestion
/**
 * @privateRemarks
 * ADO #1385: staging code changes across layers.
```